Core/Frameworks/Baikal/Core/Server.php

```diff
@@ -133,6 +133,8 @@ protected function initServer() {
 
         if ($this->authType === 'Basic') {
             $authBackend = new \Baikal\Core\PDOBasicAuth($this->pdo, $this->authRealm);
+        } elseif ($this->authType === 'Http') {
+            $authBackend = new \Sabre\DAV\Auth\Backend\Apache();
```

Comment:
Havent looked into the actual implementation, but e name makes me feel this will only work for apache?